package.json

```diff
@@ -20,6 +20,7 @@
     "analyze:copy": "npm run analyze:copy:webcomponent && npm run analyze:copy:webcomponent:src && npm run analyze:copy:demo && npm run analyze:copy:style",
     "analyze:copy:webcomponent": "copyfiles -f -e '**/*.min.js' -e '**/*.src.js' node_modules/*/vl-*.js .",
     "analyze:copy:webcomponent:src": "copyfiles -f node_modules/*/src/vl-*.js src",
+    "analyze:copy:webcomponent:dist": "copyfiles -f node_modules/*/dist/* dist",
```

Comment:
moet dat ook niet aangeroepen worden?

Reply:
Script blijkt uiteindelijk niet nodig te zijn, import probleem zat in de vl-ui-upload build dat ondertussen ook opgelost werd.